pd-control.md

```diff
@@ -491,7 +491,7 @@ export PD_ADDR=http://127.0.0.1:2379 &&
 ]
 ```
 
-### `hot [read | write | store]`
+### `hot [read | write | store| history]`
```

Comment:
I can't find any PR that adds this new command into the `pd-ctl`, it seems that it has not been developed yet?

![image](https://user-images.githubusercontent.com/1446531/134640911-6120b0a3-249e-4101-b5c0-48a167c61aff.png)

Reply:
it`s not merge yet